sample_code_submission/statistical_analysis.py

```diff
@@ -1,459 +1,83 @@
-import os
-
 import numpy as np
 from sys import path
 from systematics import systematics
 import pickle
-from iminuit import Minuit
-import matplotlib.pyplot as plt
 
 path.append("../")
 path.append("../ingestion_program")
 
 
-class StatisticalAnalysis:
+def compute_mu(score, weight, saved_info):
```

Comment:
This basicly undoes all of yuleis work.

Reply:
nothing changed by me here. Maybe I took a wrong test model